definitions/npm/moment_v2.x.x/flow_v0.25.x-/moment_v2.x.x.js

```diff
@@ -103,6 +103,8 @@ declare class moment$MomentDuration {
   days(): number;
   asDays(): number;
   months(): number;
+  asWeeks(): number;
+  weeks(): number;
```

Comment:
These were present in v2.3.x types:

https://github.com/flow-typed/flow-typed/blob/78c72eef4472905c8248ceeb51842dc912c5435a/definitions/npm/moment_v2.3.x/flow_v0.25.x-/moment_v2.3.x.js#L104-L105

Added in PR #2476